CatAnalyzer/test/run_TtbarDiLeptonAnalyzer_cfg.py

```diff
@@ -104,7 +105,8 @@
     partonTop_modes = cms.InputTag("partonTop", "modes"),
     partonTop_genParticles = cms.InputTag("partonTop"),
 
-    isTTbarMC = cms.bool(False),
+    #isTTbarMC = cms.bool(False),
+    isTTbarMC = cms.bool(runOnMC),
```

Comment:
you should make a bool for ttbarMC as not all mc requires this
and only ttbar samples have the top gen info